runtime/src/bank/tests.rs

```diff
@@ -1115,6 +1115,123 @@ fn test_distribute_rent_to_validators_overflow() {
     }
 }
 
+#[test]
+fn test_distribute_rent_to_validators_rent_paying() {
+    solana_logger::setup();
+
+    const RENT: u64 = 55;
+    const TOTAL_RENT: u64 = RENT * 2;
+
+    let rent_paying_validator = ValidatorVoteKeypairs::new_rand();
+    let rent_exempt_validator = ValidatorVoteKeypairs::new_rand();
```

Comment:
should we add a third validator that's rent-exempt going into distribution or just hope we assert it implicitly with the other tests?

Reply:
Fair enough, good idea!
No, I don't think we should assume that at all. Our test coverage mostly assumes rent is free, and validator accounts are initialized with 42 lamports 😝 
I started turning on rent everywhere by default one time, but got sidetracked before I finished.

Reply:
Done